tldw_Server_API/app/api/v1/API_Deps/ChaCha_Notes_DB_Deps.py

```
@@ -150,6 +219,81 @@ def _get_chacha_db_path_for_user(user_id: int) -> Path:
+    except Exception as e:
+        logger.warning(f"ChaChaNotes health probe failed: {e}")
+        return False
+
+
+def _create_and_prepare_db(user_id: int, client_id: str) -> CharactersRAGDB:
+    db_path: Optional[Path] = None
+    try:
+        db_path = _get_chacha_db_path_for_user(user_id)
+        try:
+            Path(db_path).parent.mkdir(parents=True, exist_ok=True)
+        except Exception as _mk2:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)